src/Illuminate/Pagination/LengthAwarePaginator.php

```diff
@@ -128,7 +128,7 @@ public function render(Presenter $presenter = null)
 
         $presenter = $presenter ?: new BootstrapThreePresenter($this);
 
-        return $presenter->render();
+        return new Expression($presenter->render());
```

Comment:
I just don't see the point in this. We have non-escaped tags for a reason. Saying they look ugly is not a reason not to use them.